docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,87 @@
+
+[discrete]
+[[setup-intelligence-page]]
+== Set up the Intelligence page
+
+Install threat intelligence integrations to add indicators to the Intelligence page.
+
+NOTE: If you've already installed threat intelligence integrations, you do not need to complete the following steps. Indicator data will automatically display within the Indicators table.
+
+. Choose one of the following:
+* From the {security-app} main menu, click *Intelligence* -> *Add Integrations*.
+* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
+. Select a threat intelligence integration, then complete the integration's guided installation.
++
+NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
+
+. Return to the Intelligence page in {elastic-sec}. Refresh the page if indicator data isn't displaying.
+
+[discrete]
+[[troubleshoot-intel-page]]
+=== Troubleshooting
+If indicator data is not appearing in the Indicators table after you installed threat intelligence integrations:
+
+* Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based the way you're collecting indicator data:
+** *{agent} integrations* - If you are collecting indicator data from {agent} threat intelligence integrations, indicator documents are stored in the `logs_ti*` index pattern.
+** *{filebeat} integrations* - If you are collecting indicator data from {filebeat} threat intelligence integrations, indicator documents are stored in the `filebeat-*` index pattern.
+* Make sure indicator data you're ingesting is mapped to {ecs-ref}[Elastic Common Schema (ECS)].
+
+[discrete]
+[[intelligence-page-ui]]
+== Intelligence page UI
+
+After you add indicators to the Intelligence page, you can <<examine-indicator-details,examine indicators>> and search, filter, and take actions on indicator data. Indicators are also visualized in the Trend view, which shows the occurrence of indicators over time.
+
+[role="screenshot"]
+image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]
+
+[discrete]
+[[examine-indicator-details]]
+=== Examine indicator details
+Learn more about an indicator by clicking *View details* and opening the Indicator details flyout. The flyout contains these informational tabs:
```

Comment:
Adding some visibility to the industry term but I'm bot sure about nuances and where to use IoC vs indicators.  
```suggestion
Learn more about an IoC by clicking *View details*, then opening the Indicator details flyout. The flyout contains these informational tabs:
```